docs/appendices/release-notes/6.0.0.rst

```diff
@@ -126,6 +126,11 @@ Scalar and Aggregation Functions
 - Added support for the :ref:`aggregation-stddev-pop` function to compute
   the population standard deviation.
 
+- Updated the default ``compression`` of
+  :ref:`percentile aggregation <aggregation-percentile>` from ``100.0`` to
+  ``200.0`` which increases the accuracy of the approximations but with the
+  cost of more memory consumptions and longer execution time.
```

Comment:
This reads a bit scary, maybe we can soften this by:
```suggestion
  ``200.0`` which increases the accuracy of the approximations but with the
  cost of slightly more memory consumption and increased execution time,
  depending on the used data set.
  To regain existing behaviour, adjust the ``compression`` argument of the
  :ref:`percentile aggregation <aggregation-percentile>` to your needs.
```

Additionally, I think this is a breaking change.
Shouldn't we also mention something about the changed algorithm? Maybe under the performance section?
E.g.
Replaced the `t-digest <https://github.com/tdunning/t-digest>`_ algorithm used by the 
:ref:`percentile aggregation <aggregation-percentile>` to improve the consistency and 
accuracy of the result.

Reply:
Thank you, moved the current one to breaking changes and added below to the regular changes section

> Replaced the t-digest <https://github.com/tdunning/t-digest>_ algorithm used by the
:ref:percentile aggregation <aggregation-percentile> to improve the consistency and
accuracy of the result.